### PR TITLE
 search: allow to customize the limit

### PR DESCRIPTION
**Pull request by maxlath**
before those changes, whatever the request limit, only 20 results where taken from ElasticSearch

### CODE REVIEW

File: tests/api/search/search.test.coffee [hunk 1]
```diff
@@ -37,18 +37,7 @@ describe 'search:global', ->
     .then undesiredRes(done)
     .catch (err)->
       err.statusCode.should.equal 400
-      err.body.status_verbose.should.equal 'invalid type: da'
-      done()
-    .catch undesiredErr(done)
-
-    return
-
-  it 'should reject search without lang', (done)->
```
Comment (jum-s) [inline tests/api/search/search.test.coffee]: how is this test not relevant anymore?

Comment (maxlath) [reply tests/api/search/search.test.coffee]: the use of sanitize [makes the `lang` parameter default to `en`](https://github.com/inventaire/inventaire/blob/daa39bc/server/lib/sanitize/parameters.coffee#L111), so 3 options:
* remove the spec to throw on a missing lang parameter (the one currently implemented)
* customize the parameter config to request that a missing `lang` be rejected
* change the parameter general behavior, but that might impact other places depending on it

Comment (jum-s) [reply tests/api/search/search.test.coffee]: what about testing the default lang behavior ?
`should return a english lang label if no language parameter is passed`
